docs/resources/er_route_table.md

```diff
@@ -0,0 +1,119 @@
+---
+subcategory: "Enterprise Router (ER)"
+---
+
+# huaweicloud_er_route_table
+
+Manages a route table resource under the ER instance within HuaweiCloud.
+
+## Example Usage
+
+```HCL
+variable "instance_id" {}
+variable "route_table_name" {}
+
+resource "huaweicloud_er_route_table" "test" {
+  instance_id = var.instance_id
+
+  name        = var.route_table_name
```

Comment:
```suggestion
  instance_id = var.instance_id
  name        = var.route_table_name
```
I think it's better to remove the blank line

Reply:
done.